ecmascript/minifier/tests/compress/fixture/issues/2262/1/input.js

```diff
@@ -0,0 +1,20 @@
+(() => {
+    "use strict";
+
+    var commonjsGlobal = globalThis
+
+    function createEventEmitter(value) {
+    }
```

Comment:
Removing this `createEventEmitter` or `commonjsGlobal` (both of which are unused) will cause the bug not to reproduce. No idea why, though